{{cookiecutter.repo_name}}/package.json

```diff
@@ -106,7 +106,7 @@
         "git add"
       ],
       "**/!(definitions)/*.css": [
-        "prettier --write",
+        "prettier --write --single-quote",
```

Comment:
This should just be `prettier-standard --write` The same as the `*.js` one.

Reply:
I thought `prettier-standard` was a Javascript-specific formatter.

Reply:
Nah, `prettier-standard` is just `prettier` but with all the rules of StandardJS (Like single quotes)